test/e2e/federated-service.go

```diff
@@ -34,6 +35,9 @@ const (
 
 	FederatedServiceName    = "federated-service"
 	FederatedServicePodName = "federated-service-test-pod"
+
+	KubeDNSConfigMapName      = "kube-dns"
```

Comment:
This is the default value of configmap name that kube-dns looks for. To change the name here, will also need to update the config map name flag passed to kube dns.
Leaving it as is